kornia/geometry/transform/affwarp.py

```diff
@@ -668,6 +668,10 @@ class Resize(nn.Module):
         >>> out = Resize((6, 8))(img)
         >>> print(out.shape)
         torch.Size([1, 3, 6, 8])
+
+    .. raw:: html
```

Comment:
I did this so that the demo would be tied to the particular function when it's rendered in the docs. Otherwise if this gets moved then the docs could be out of sync. I could also create a custom if you think that would be nice, or I can undo this particular change.